### ISSUE TITLE
Where are my STX?

### ISSUE THREAD

**Issue by DesertSon** [blockstack/stacks-wallet#559]
Hello,
Hoping someone can help me with a STX transaction issue. First, let me say I don't code or anything close to it. And reading transactions on the blockchain find me in over my head pretty quickly. I arrived here by searching the Stacks Wallet website. My issue is that on January 15 I tried to move my STX off of KuCoin and send it to my Stacks Wallet. That transaction failed. KuCoin Support said it was a network issue and to try again. I tried again on January 20. KuCoin reports that the withdrawal was successful but now 2 days later I still haven't received my STX. I've checked and rechecked my STX wallet deposit address, it's correct. I see that Stacks wallet says KuCoin and other exchanges have had an issue with Stacks 2.0. I informed KuCoin of this but they reply if the transaction shows complete then I need to inquire with Stacks wallet folks. I feel a bit stuck between both parties. I'm wondering where my STX are since they no longer appear in my KuCoin account and don't appear in my Stacks wallet. BTW, I do see that the Stacks wallet requires a small amount of BTC to send STX but I'm receiving into the wallet not sending out. Nonetheless if it's important to my issue, there is no BTC in the wallet. Any suggestions from folks more adept at this stuff than me would be greatly appreciated. Thank you in advance for considering my issue.

**Comment by aulneau** [comment]
Hi there! Sorry to hear about your troubles! Can you confirm which version of the stacks wallet you are using? From what you've written, it seems you are using an older version of the wallet. You can find the latest version here, https://github.com/blockstack/stacks-wallet/releases/tag/v4.0.3 you will need your original seed phrase / secret key (a sentence of 12 or 24 words).

**Comment by DesertSon** [comment]
Hi Thomas,

First, thank you so much for responding. I followed the link you provided
and comparing that version to my wallet showed me I did indeed have an
older wallet version.  After installing the newer version - there was my
full deposit showing in the wallet. Looking back it seems like such a
logical thing to do, that is, check my wallet version but I just assumed I
had the most current version. Well, a lesson learned. Thank you again
Thomas!
Sincerely, Ben
